eahub/profiles/index.py

```diff
@@ -51,6 +50,49 @@ class ProfileIndex(AlgoliaIndex):
             ["looking_for", "looking_for"],
         ]
 
+    class ProfileInternalIndex(AlgoliaIndex):
+        index_name = settings.ALGOLIA["INDEX_NAME_PROFILES_INTERNAL"]
+        should_index = "is_searchable_internal"
+        fields = ProfilePublicIndex.fields
+
+    @register(Profile)
+    class ProfileMetaIndex(AlgoliaIndex):
+        def __init__(self, model, client, settings):
+            self.indices = [
+                ProfilePublicIndex(model, client, settings),
+                ProfileInternalIndex(model, client, settings),
+            ]
```

Comment:
I saw that it's part of the docs, and of course following the official advice would be ideal. `algoliasearch_django` is a special case though, I've spent few months studying its codebase, and now I'm very hesitant to apply something that is listed as a best practice there 😅

Reply:
Do you mean having two different Profile classes, one for internal, one for public? Wouldn't that great quite a bit of overhead, e.g., you'd need 2 tables? And wouldn't having more than 1 profile model create maintainability issues?

I've just pushed some code with a naive first implementation of what I think you have in mind, but it didn't work because the "ProfilesInternal" table is empty.

Reply:
ok, the error makes sense